template.tpl

```diff
@@ -318,6 +318,43 @@ ___TEMPLATE_PARAMETERS___
         "type": "EQUALS"
       }
     ]
+  },
+  {
+    "type": "GROUP",
+    "name": "additionalInformation",
+    "groupStyle": "ZIPPY_CLOSED",
+    "subParams": [
+      {
+        "type": "SIMPLE_TABLE",
+        "name": "products",
+        "displayName": "Product Information",
+        "simpleTableColumns": [
+          {
+            "defaultValue": "",
+            "displayName": "Product ID",
+            "name": "id",
+            "type": "TEXT",
+            "valueHint": ""
+          },
+          {
+            "defaultValue": "",
+            "displayName": "Product Name",
+            "name": "name",
+            "type": "TEXT",
+            "valueHint": ""
+          },
+          {
+            "defaultValue": "",
+            "displayName": "Product Category",
+            "name": "category",
+            "type": "TEXT",
+            "valueHint": ""
+          }
+        ],
+        "help": "Configure product-level information with Reddit. Use one row per unique product. The Product ID and Product Category are required, but the Product Name is optional."
```

Comment:
sn00bie question: the category is required?  I would've figured it'd be optional, and that the only required field is the ID.

Reply:
Yeah a category is required because we don't want to ingest any information that is potentially sensitive. If we put forth that we expect the advertisers to be truthful, by filtering out anything with a sensitive category, we absolve ourselves of liability of ingesting sensitive information.

For Alpha, it's technically not required because we'll be working with non-sensitive advertisers, but I'd rather just do it once than special-case it for Alpha.

Reply:
Ahhhh, I totally missed that this is just the help text.  👍   Nevermind me.